doc/diffview.txt

```diff
@@ -73,10 +74,13 @@ Example configuration with default settings:
         win_config = {    -- See |diffview-config-win_config|
           position = "bottom",
           height = 16,
+          win_opts = {}
         },
       },
       commit_log_panel = {
-        win_config = {},  -- See |diffview-config-win_config|
+          win_config = {    -- See ':h diffview-config-win_config'
```

Comment:
Please use `|diffview-config-win_config|` here so that users can jump directly to the tag with `:h CTRL-]`.